### PR TITLE
feat: add token network icon

### PR DESCRIPTION
**Pull request by bonustrack**
This PR adds network icon on the treasury tokens and send token modal, also sort token by balance
image

**How to test**

- Go here:
http://localhost:8080/#/s:balancer.eth/treasury
http://localhost:8080/#/matic:0x80d0ffd8739eabf16436074ff64dc081c60c833a/treasury
or another space with treasury.
- Try send token with token picker

### CODE REVIEW

File: apps/ui/src/components/Ui/TokenStamp.vue [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
```
Comment (Sekhmet) [inline apps/ui/src/components/Ui/TokenStamp.vue:1]: I think it should be called `StampToken` based on [naming convention](https://vuejs.org/style-guide/rules-strongly-recommended#order-of-words-in-component-names).

@samuveth

File: apps/ui/src/components/Ui/TokenStamp.vue [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { getNetwork } from '@/networks';
+import { NetworkID } from '@/types';
+import { getUrl } from '@/helpers/utils';
+
+const props = withDefaults(
+  defineProps<{
+    id: string;
+    size?: number;
+    width?: number;
+    height?: number;
```
Comment (Sekhmet) [inline apps/ui/src/components/Ui/TokenStamp.vue:11]:
```suggestion
```

Those are not used.

File: apps/ui/src/components/Ui/TokenStamp.vue (hunk 1 above)
Comment (samuveth) [reply apps/ui/src/components/Ui/TokenStamp.vue:1]: Yes, `StampToken`

**Comment by bonustrack** [comment]
Thanks for the review just fixed

**Pull request merged**